pydrad/parse/parse.py

```diff
@@ -397,6 +403,17 @@ def _read_hstate(self):
         for col in self._hstate_data.colnames:
             _add_property(col, '_hstate_data')
 
+    def _read_scl(self):
+        """
+        Parse the time-scale (``.scl``) file and set the relevant attributes.
+        """
+        if not self._scl_filename.is_file():
+            log.warning(f'{self._scl_filename} not found.  Skipping parsing of .scl files.  Set read_scl=False to suppress this warning.')
```

Comment:
```suggestion
            log.warning(f'{self._scl_filename} not found.  Skipping parsing of .scl files.  Set read_scl=False to suppress this warning.')
            return 
```

Reply:
Did we have a test that was failing when this was missing?

Reply:
No, not for this one, but codecov did complain the line wasn't covered.